scripts/kind-e2e/lib_subctl_gather_test.sh

```diff
@@ -11,7 +11,7 @@ function test_subctl_gather() {
   rm -rf $out_dir
   mkdir $out_dir
 
-  ${DAPPER_SOURCE}/bin/subctl gather --dir $out_dir
+  subctl gather --dir $out_dir
```

Comment:
This tests `subctl gather ` and should also be removed.

Reply:
The whole file should probably be removed?